statefun-flink/statefun-flink-core/src/test/resources/module-v2_0/module.yaml

```diff
@@ -37,6 +37,10 @@ module:
               - name: seen_count
                 expireAfter: 60000millisecond
                 expireMode: after-invoke
+            timeout: 2minutes
+            connectTimeout: 20seconds
+            readTimeout: 1second
```

Comment:
Can the example say 10second on the read timeout?
The reason is that the examples are usually copy-pasted.